packages/cdktf-cli/lib/get/generator/resource-parser.ts

```diff
@@ -291,6 +291,7 @@ class Parser {
               struct,
               isOptional: optional,
               isRequired: required,
+              isSingleItem: true,
```

Comment:
This isn't actually a list, so this value won't be used. Doesn't hurt anything though.